src/Events/Custom_Tables/V1/Tables/Occurrences.php

```diff
@@ -60,14 +60,15 @@ protected function get_update_sql() {
 		$table_name      = self::table_name( true );
 		$charset_collate = $wpdb->get_charset_collate();
 
+		// VARCHAR(19) to store YYYY-MM-DD HH:MM:SS values as strings and allow partial compare.
```

Comment:
For partial compares is this because doing date comparisons isn't ideal, i.e. `select start_date from wp_tec_occurrences where DATE(start_date) = '2022-12-01' ` ?

Reply:
The `DATE` comparison **is** ideal, and in a world where back-compatibility is not an issue, I would stick with it. The problem is existing TEC and other code that is assuming those fields are strings.